### DISCUSSION TITLE
node activity vizualizer

### DISCUSSION THREAD

**Discussion by caitlindurkee** [bonsai-rx/bonsai#979] [Q&A]
Hello, I'm fairly new to Bonsai and github, so apologies if this has been answered here previously or documented elsewhere, but I'm wondering if there is a way to individually scale the y axis for different real-time traces in the node vizualizer. e.g. I'm looking at fiber photometry signals from 4 mice running at the same time, so I have 0G-3G signals on my screen. Some of the signals are smaller amplitude than others and I want to be able to specifically zoom in (increase yaxis scale) on those particular traces. Is there a way to do that? 
Thanks!

**Comment by bruno-f-cruz** [comment]
Hi, 
Welcome to Bonsai!

What is the data type of your photometry recordings? Might be easier if you add an workflow.
Thanks!

**Reply by caitlindurkee** [reply]
Hi Bruno,

Thanks for the quick reply! Attached is a screen shot of the workflow. In the screenshot below, I would double click the L470 to see the real-time traces of my photometry signals. What do you mean by data type? The output is an excel file of the photometry signals, which technically are pixel brightness values.
![Screenshot 2022-07-27 131337](https://user-images.githubusercontent.com/109764695/181364000-c6792e38-be44-4642-9b0d-27e29c044a17.png)

**Comment by bruno-f-cruz** [comment]
You can check the datatype by right clicking on top of a node and checking its output.

Additionally, can you attach the actual workflow file (*.bonsai)? Sorry, but I am not familiar with this package. 

Looking at the github repo, it looks like there is a [Neurophotometrics.Design](https://github.com/neurophotometrics/neurophotometrics/tree/master/Neurophotometrics.Design) package that should be installed, can you check if you have it installed?

Finally, if your issue is specific to this package you might have more luck opening an issue directly in [their repo](https://github.com/neurophotometrics/neurophotometrics/issues).

Hope it helps!

**Reply by caitlindurkee** [reply]
Hello,

I will take a look at those repos - thanks! And I will check if we have that design package installed. I do know that folks in the lab have indeed been able to click the activity vizualizer to zoom in on the axes for particular traces, but it has all of a sudden stopped having that function. Any thoughts on this?
Regarding the workflow .bonsai file - github says it doesn't support that file type, so I'm not able to attach it, unfortunately. 

Thanks!

Caitlin
